src/giftCards/GiftCardUpdate/GiftCardHistory/GiftCardHistory.tsx

```diff
@@ -88,7 +88,8 @@ const GiftCardHistory: React.FC = () => {
                       user={user}
                       message={message}
                       key={id}
-                      hasPlainDate
+                      app={app}
+                      hasPlainDate={false}
```

Comment:
question: why did you change that, having there plain date was intended

Reply:
How do I know if it's intended? I don't think giving user an exact date and time is better than "x days ago" (full date is available on hover).